### PR TITLE
feat: add md reading notice

### PR DESCRIPTION
**Pull request by rabbyOnTheMars**
To do: 
before next release, will delete the test md file, and reorganize the file structure

### CODE REVIEW

File: changeLogs/index.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import version0161 from './test0161.md';
+const version = process.env.version!.split('/')[0].replace(/[^0-9]/g, '');
+const versionMap = {
+  '0161': version0161,
+  '0167': version0161,
```
Comment (vvvvvv1vvvvvv) [inline changeLogs/index.ts:5]: mock data for test should not merge into master, just remove them after test passed

File: src/background/service/preference.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { keyringService, sessionService, i18n } from './index';
 import { TotalBalanceResponse, TokenItem } from './openapi';
 import { HARDWARE_KEYRING_TYPES } from 'consts';
 import { browser } from 'webextension-polyfill-ts';
-
+const version = process.env.version!.split('/')[0].replace(/[^0-9]/g, '');
```
Comment (vvvvvv1vvvvvv) [inline src/background/service/preference.ts:7]: try `process.env.release`, no time string and extra text

File: src/background/service/preference.ts [hunk 3]
```diff
@@ -243,6 +247,19 @@ class PreferenceService {
   updateWalletSavedList = (list: []) => {
     this.store.walletSavedList = list;
   };
+  getIsFirstOpen = () => {
+    if (
+      !this.store.currentVersion ||
+      version > (this.store.currentVersion || 0)
```
Comment (vvvvvv1vvvvvv) [inline src/background/service/preference.ts:253]: here we convert version into number for compare may cause a issue: when compare between 0161 and  100, 0161 is greater but actually `1.0.0` should be greater
how about just keep the point like `0.16.1` and use package like https://www.npmjs.com/package/compare-versions for compare?

**Review by vvvvvv1vvvvvv — CHANGES_REQUESTED** [review]

File: src/background/service/preference.ts (hunk 2 above)
Comment (rabbyOnTheMars) [reply src/background/service/preference.ts]: done

File: src/background/service/preference.ts (hunk 3 above)
Comment (rabbyOnTheMars) [reply src/background/service/preference.ts]: done

File: changeLogs/index.ts (hunk 1 above)
Comment (rabbyOnTheMars) [reply changeLogs/index.ts]: Yes, you are right, will add form notice before merge into master

**Review by vvvvvv1vvvvvv — APPROVED** [review]

**Pull request merged**